public/locales/pt/common.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "O modo mais fácil de criar um",
+  "readme": "README",
+  "description": "O nosso simples editor permite que você facilmente adicione e personalize todas as seções que você precisa para o README do seu projeto",
+  "get-started": "Comece aqui",
+  "100-free": "100% gratuito",
+  "made-with-love": "Feito com ",
+  "by": " por",
+  "language": "Idioma",
+  "english": "Inglês",
+  "french": "Francês",
+  "portuguese": "Português",
+  "spanish": "Espanhol",
+  "italian": "Italiano",
```

Comment:
Heads-up: these entries were [removed in #96](https://github.com/katherinepeterson/readme.so/pull/96/files#diff-4840952983d2a94f27deb00bf7247347345b60e3a96dc2aa41a6a50ecacdf1b6).

Reply:
Ops, going to remove it haha